### ISSUE TITLE
Build the site on PRs

### ISSUE THREAD

**Issue by e-minguez** [suse-edge/suse-edge.github.io#442]
Our friends from the doc team do it like this https://github.com/SUSE/release-notes-alp/blob/main/.github/workflows/asciidoc.yml

**Comment by hardys** [comment]
This alignment would be great - I did look into this a while ago but ran into issues getting our docs to build with the opensuse daps containers, so perhaps as a first step we could switch from the container @agracey built to that (I didn't have time to investigate the issues unforunately)

**Comment by hardys** [comment]
> [@hardys](https://github.com/hardys) Is this still an issue?

@fsundermeyer yes I think it is - if you see e.g https://github.com/suse-edge/suse-edge.github.io/blob/main/.github/workflows/test-deploy.yml#L25 we're still using a special image from @agracey which works, but IIUC isn't aligned with other product docs?

It would be good to review the workflows we're using and where possible adjust to ensure we're fully aligned with what other teams are doing and the downstream docs tooling.
